gwctl/pkg/utils/types.go

```diff
@@ -100,13 +100,16 @@ func MustPolicyManagerForTest(t *testing.T, fakeClients *common.K8sClients) *pol
 type OutputFormat string
 
 const (
+	OutputFormatWIDE  OutputFormat = "wide"
```

Comment:
```suggestion
	OutputFormatWide  OutputFormat = "wide"
```
As wide is not an acronym for something but an actual word, we should keep it `CamelCase`.